README.md

```diff
@@ -21,6 +21,8 @@ Deployments with Sphinx are a three-step process:
 
 * **Compatible with Forge Scripts**: You can integrate Sphinx by adding a few lines of code to your existing Forge scripts.
 
+* **No Lock-In**: You can execute deployments from your local machine without using the Sphinx DevOps Platform. These deployments are executed in the exact same manner as they would be executed with the DevOps Platform, including identical contract addresses.
```

Comment:
@RPate97 Curious to hear your thoughts on this bullet point

Reply:
Appended a new sentence to this bullet point: " This includes networks that the DevOps Platform doesn't currently support."

I also added a note in the "Supported Networks" section since I agree it's relevant there too

> Also one question: Do you think we should list that this only works for projects with a single owner in the current limitations?

Updated the "Current Limitations" section